vespa-application-maven-plugin/src/main/java/com/yahoo/container/plugin/mojo/ApplicationMojo.java

```diff
@@ -93,6 +93,10 @@ private void addBuildMetaData(File applicationDestination) throws MojoExecutionE
         }
     }
 
+    static boolean isVespaParent(String groupId) {
+        return groupId.matches("(com\\.yahoo\\.vespa|ai\\.vespa)(\\..+)?");
```

Comment:
Add `^` to avoid matching with example.com.ai.vespa?

Reply:
Urgh, yes, it was "startsWith" earlier.

Reply:
It broken because the trailing `.` was added ...